library/src/pivotal-ui-react/radio/radio.js

```diff
@@ -10,16 +10,26 @@ var Radio = React.createClass({
     onChange: React.PropTypes.func,
     id: React.PropTypes.string,
     className: React.PropTypes.string,
-    style: React.PropTypes.object
+    style: React.PropTypes.object,
+    disabled: React.PropTypes.bool
   },
 
   render: function() {
-    const {className, style, children, ...others} = this.props;
+    const {className, style, children, disabled, ...others} = this.props;
     const props = mergeProps({className: className, style: style}, {className: 'radio'});
+
     return (
       <div {...props}>
-        <label>
-          <input type="radio" {...others}/>
+        <label
+          className={disabled ? 'type-neutral-5' : ''}
+          style={disabled ? {cursor: 'not-allowed'} : undefined}
```

Comment:
Do you really want `cursor: 'not-allowed'` here? Would `cursor: 'default'` be good enough? You can tell a label is active because it has `cursor: pointer` Also, my advice on slack about multiple classes was more general advice for your application. In this case I would suggest a style for label.disabled that has both the color and the cursor so that it is easier for people to override the color if they really wanted to.